### PR TITLE
53 modified GSHA-based advisories

### PR DESCRIPTION
**Pull request by jasnow**
Yes, large number of modified files (mainly from github_advisory_sync.rb script):
  * Added cvss_v3: and ghsa: fields.
  * Removed unused unaffected_versions: fields.
  * Moved osvdb: fields.
  * Changed single to double quotes for patched_versions: fields.
  
Then
  * My script did these fixes:
 ```
 description |
 ```
  * I manually undid single quotes around title: fields and some line wrapping.
  
  yamllint and rake (green)

### CODE REVIEW

File: gems/ruby_parser/CVE-2013-0162.yml [hunk 1]
```diff
@@ -2,12 +2,14 @@
 gem: ruby_parser
 cve: 2013-0162
 osvdb: 90561
+ghsa: 8mvw-22r7-w6fq
 url: https://nvd.nist.gov/vuln/detail/CVE-2013-0162
-title: "CVE-2013-0162 rubygem-ruby_parser: incorrect temporary file usage"
+title: "CVE-2013-0162 rubygem-ruby_parser: Incorrect temporary file usage"
```
Comment (postmodern) [inline gems/ruby_parser/CVE-2013-0162.yml:7]: Does this really need to be capitalized?

File: gems/ember-source/CVE-2013-4170.yml [hunk 2]
```diff
@@ -13,13 +13,13 @@ description: |
   application assigns a view's `tagName` to user-supplied data, a
   specially-crafted payload could execute arbitrary JavaScript in the
   context of the current domain ("XSS").
-
```
Comment (postmodern) [inline gems/ember-source/CVE-2013-4170.yml:16]: I think the newline helps add emphasis.

File: gems/fog-dragonfly/CVE-2013-5671.yml [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
 osvdb: 96798
+ghsa: qrgf-jqqm-x7xv
 url: https://nvd.nist.gov/vuln/detail/CVE-2013-5671
 title: fog-dragonfly Gem for Ruby imagemagickutils.rb Remote Command Execution
 date: 2013-09-03
 description: |
   fog-dragonfly Gem for Ruby contains a flaw that is due to the program
   failing to properly sanitize input passed via the imagemagickutils.rb script.
   This may allow a remote attacker to execute arbitrary commands.
-
```
Comment (postmodern) [inline gems/fog-dragonfly/CVE-2013-5671.yml:12]: I think the newline helps add emphasis. Maybe add a "Note:" to the last line?

**Review by postmodern — COMMENTED** [review]
Noticed a few formatting things and had questions.

File: gems/ember-source/CVE-2013-4170.yml (hunk 2 above)
Comment (jasnow) [reply gems/ember-source/CVE-2013-4170.yml:16]: Added newline back

File: gems/fog-dragonfly/CVE-2013-5671.yml (hunk 3 above)
Comment (jasnow) [reply gems/fog-dragonfly/CVE-2013-5671.yml:12]: Add newline back

**Review by postmodern — APPROVED** [review]

**Pull request merged**